src/components/NavTabs.js

```diff
@@ -10,7 +11,7 @@ const NavTabs = () => {
           to add item
         </NavLink>
 
-        <NavLink className="right-tab" to="/list">
+        <NavLink className="right-tab" to={`/list/${getCurrentToken()}`}>
```

Comment:
Again, I'm confused why we're now keeping the token in the URL instead of just accessing it via `localStorage`?